uat/conftest.py

```diff
@@ -1,7 +1,24 @@
 import pytest
 
+from t_setup import GdkProcess, GdkInstrumentedProcess
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        "--gdkimpl", action="store", default="installed", help="supports two gdk implementations: installed or code"
```

Comment:
Use action = "store_true" and remove default. Then the coverage commands can be reduced to 
`coverage run --source=gdk -m pytest -v -s uat --gdkimpl  && coverage xml --fail-under=77` and 
`coverage run --source=gdk -m pytest -v -s uat && coverage xml --fail-under=77`

Reply:
So basically remove the value, the infer from existence of the parameter. Ok!